apps/emqx/src/emqx_misc.erl

```diff
@@ -533,6 +547,14 @@ readable_error_msg(Error) ->
             end
     end.
 
+try_to_existing_atom(Fun, Args) ->
+    try erlang:apply(Fun, Args) of
```

Comment:
avoid use `erlang:apply/2`, you can try .. catch in L469

Reply:
fix in #9285